docs/source/support/release-notes.rst

```diff
@@ -11,6 +11,26 @@ HEAD — ongoing
 
 - Sandbox: Transactions with a record time that is after the maximum record time (as provided in the original command)
   are now properly rejected instead of committed to the ledger: `#987 <https://github.com/digital-asset/daml/issues/987>`_
+- **BREAKING** Ledger API: Removed the unused field :ref:`com.digitalasset.ledger.api.v1.ExercisedEvent` from :ref:`com.digitalasset.ledger.api.v1.Event`.
```

Comment:
Could you point out that the format is still wire-compatible -- that is, old clients will still work with new servers.

Reply:
Done. See ed7f246